content/applications/finance/accounting/payables/misc/forecast.rst

```diff
@@ -2,71 +2,49 @@
 Forecast future bills to pay
 ============================
 
-When you get hundreds of vendor bills per month with each of them having
-different payment terms, it could be complex to follow what you have to
-pay and when. Paying your vendors too early can decrease your cash
-availabilities and paying too late can lead to extra charges.
-
-Fortunately, Odoo provides you the right tools to manage payment orders
-to vendors efficiently.
+In Odoo you have the possibility to manage payments by setting automatic :guilabel:`Payments Terms`
+and :guilabel:`follow-ups`.
 
 Configuration: payment terms
 ============================
 
-In order to track the vendor conditions, we use payment terms in Odoo.
-Payment terms allow to keep track of the conditions to compute the due
-date on an invoice. As an example, a payment term can be:
+In order to track the vendor conditions, we use :guilabel:`Payment Terms` in Odoo.
+:guilabel:`Payment terms` allow keeping track of due dates on invoices. Examples of
+:guilabel:`Payment Terms` are:
 
 -  50% within 30 days
-
 -  50% within 45 days
 
-To create your most common payment terms, use the menu: :menuselection:`Configuration -->
-Management --> Payment Terms` in the **Accounting** application. The following
-example show a payment term of 30% directly and the balance after 30
-days.
-
-.. image:: forecast/forecast01.png
-  :align: center
-
-Once payment terms are defined, you can assign them to your vendor by
-default. Set the Vendor Payment Term field on the Accounting tab of a
-vendor form. That way, every time you will purchase to this vendor, Odoo
-will propose you automatically the right payment term.
+To create :guilabel:`Payment terms`, use the :menuselection:`Accounting app --> Configuration -->
+Invoicing --> Payment Terms`. By clicking on :guilabel:`Create`, you can add the terms you need as
+well as modify existing ones by clicking on it in the Payment Terms list in the same view.
 
```

Comment:
I think it could be fine to explain more about "how to create", like what the term types are, the due date computation
![image](https://user-images.githubusercontent.com/95279628/202469472-8b586b10-1c52-4005-ac77-7e4eaaf6b8a3.png)

Reply:
Also, would an image here be a good idea? What do you think? I know it looks quite simple in v14.0, but way more complex in Odoo16

Reply:
I won't get into detail as it can be confusing to explain dates. We already have a video that explains it and even there we had a lot of confused people commenting. So I might just link the video instead

Reply:
Great idea :)